test/e2e/app-dir/rewrite-headers/rewrite-headers.test.ts

```diff
@@ -315,7 +315,8 @@ const cases: {
       RSC: '1',
     },
     expected: {
-      'x-nextjs-rewritten-path': null,
+      // Vercel matches `/` to `/home`
+      'x-nextjs-rewritten-path': '/home',
```

Comment:
vercel.com now matches `/home` (`x-matched-path`). I don't know why this is surfaced in our API. Feels like this is leaking implementation details of the other Next.js page we're redirecting to. 

But if it's a bug, it was always there and only surfaced after vercel.com started matching `/` to `/home` internally.

Maybe this should test against `x-middleware-rewrite` instead?